utils.go

```diff
@@ -25,3 +29,29 @@ func newFile(name string) (*os.File, error) {
 
 	return file, err
 }
+
+func getJSON(url string, data interface{}) error {
+	res, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+
+	defer res.Body.Close()
+	if res.StatusCode != 200 {
+		return errors.New("invalid status code")
+	}
+
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return err
+	}
+
+	// fmt.Printf("%s", body)
+
+	err = json.Unmarshal(body, &data)
```

Comment:
This and the following lines can be resumed in:

```golang
return json.Unmarshal(body, &data)
```

Reply:
👍